src/content-script.ts

```diff
@@ -4,13 +4,40 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+const regexPathName = /\/.+\/.+\/(issues|pull)\/\d+/;
+const footprint = "refined-saved-replies";
+
+function addObserver(callback: MutationCallback) {
+	const options = { childList: true, subtree: true };
+
+	const observer = new MutationObserver(callback);
+
+	observer.observe(document.body, options);
+}
+
+function verifyLocation() {
 	// 1. Is this an issue I can reply to?
 	// (if not, exit from the page)
 	const replySummary = document.getElementById("saved-reply-new_comment_field");
-	if (!replySummary) {
-		return;
+
+	if (
+		regexPathName.test(window.location.pathname) &&
+		!!replySummary &&
+		!replySummary.hasAttribute(footprint)
```

Comment:
[Nitpick] Simplifying this a bit:

```suggestion
		!document.getElementById("saved-reply-new_comment_field")?.hasAttribute(footprint)
```

...and while you're here, if `footprint` makes sense to store in a reused variable (+1), how about `saved-reply-new_comment_field` too?

Reply:
Nice catch! I think I can simplify more 🤔 `saved-reply-new_comment_field` only needs to be set once

Thanks for bringing this to my attention!

Reply:
@JoshuaKGoldberg I don't mind if you make changes! I'm the student here 😄

Reply:
I guess what I meant is - it's better if you make them (I like giving folks the pleasure of owning their stuff!). But if you don't have time, explicitly let me know and I can take over.

Anyway, you can re-request my review whenever you're ready. ❤️